src/mesh/config/DataConfiguration.cpp

```diff
@@ -63,7 +63,7 @@ void DataConfiguration::xmlTagCallback(
     int         dataDimensions = getDataDimensions(typeName);
     addData(name, dataDimensions);
   } else {
-    PRECICE_ERROR("Received callback from tag " << tag.getName());
+    PRECICE_ASSERT(false, "Received callback from unknown tag " << tag.getName());
```

Comment:
Not sure how to test this, but it is already a bit clearer than before. I guess that an assertion is a good idea here, as it should be caught earlier. :+1: